docs/releases/v2020.1.3.rst

```diff
@@ -0,0 +1,58 @@
+Gluon v2020.1.3
+===============
+
+Bugfixes
+--------
+
+- Fixes a bug in musl which can lead to spurious crashes in fastd and other programs, which frequently change from single to multi-threaded operation.
```

Comment:
"alternate between single- and multi-threaded operation" (the actual bug affected the change from multi- to single-threaded)